Modules/Module_Processing/Module_Population/xASL_adm_GzipAllFiles.m

```diff
@@ -54,7 +57,11 @@ function xASL_adm_GzipAllFiles(ROOT, bFolder, bUseLinux, pathExternal)
 		if exist(ROOT,'dir') % Check if directory exists, otherwise a random current directory would be zipped
 			PathToSearch = xASL_adm_UnixPath(ROOT);
             oldPath = pwd;
-			[exit_code, system_result] = system(['cd ' PathToSearch '; for i in `find * | grep -E \.nii$`; do gzip -1 -f -q -v "$i"; done'], '-echo');
+			if bVerbose
```

Comment:
Is removing the -v option sufficient, as we still have the -echo option? (I mean did you test it)?